modules/tinymce/src/core/main/ts/Deprecations.ts

```diff
@@ -48,7 +48,7 @@ const logRemovedWarnings = (rawOptions: RawEditorOptions, normalizedOptions: Nor
     // eslint-disable-next-line no-console
     console.warn(
       'The following deprecated features are currently enabled and have been removed in TinyMCE 6.0. These features will no longer work and should be removed from the TinyMCE configuration. ' +
-      'See https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/ for more information.' +
+      'See https://www.tiny.cloud/docs/tinymce/6/6_0_0-release-notes-overview.html for more information.' +
```

Comment:
This should probably be a migration guide, not the release notes, but I guess the release notes will include a link to the guide 🤔

Reply:
Agreed, this should link to the migration guide instead.

- [ ] Use the migration guide link instead